### DISCUSSION TITLE
Pls Support with file and folder have onedot first name

### DISCUSSION THREAD

**Discussion by nightrelax** [qkqpttgf/OneManager-php#699] [General]
Hi admin,
i install project in Vercel with onedrive
My data have file and folder with name . in first name
In manager show file and folder have . in first name
but guest dont show
pls support for my name data

and add a mini quick search file name to top right

**Comment by qkqpttgf** [comment]
yes, a file/folder start with . is hide.
It is feature, you can hide something to guest.
